docs/overview/glossary.rst

```diff
@@ -29,11 +29,11 @@ All of these new words can get confusing! We've created a glossary to help nail
     Ivy Compiler
         A wrapper function around native compiler functions, which uses lower level compilers such as XLA to compile to lower level languages such as C++, CUDA, TorchScript, etc.
 
-    Graph Compiler
-        Graph compilers map the high-level computational graph coming from frameworks to operations that are executable on a specific device.
+    Tracer
```

Comment:
But this is still called "Tracer" in other frameworks as well. Like `tf.function` and `torch.jit.trace` both are called tracers rather than compilers.

Reply:
I'd argue that they are graph compiler more generally, i.e. `torch.compile`, `torch.jit.script`, etc. Some of them are tracers but I feel like Graph compiler is more general :thinking: